.github/workflows/release.yml

```diff
@@ -64,6 +64,17 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v4
 
+      - name: Determine tag from myst-to-react package.json
+        id: get_tag
+        run: |
+          VERSION=$(jq -r '.version' packages/myst-to-react/package.json)
+          if [ -z "$VERSION" ]; then
+            echo "Unable to read version from packages/myst-to-react/package.json" >&2
+            exit 1
+          fi
+          TAG="myst-theme@$VERSION"
```

Comment:
after conversation in Discord switched this to myst-theme

so it reads the version from myst-to-react but then it calls the release myst-theme